src/app/dashboard/dashboard.component.ts

```diff
@@ -79,18 +77,18 @@ export class DashboardComponent implements OnInit {
    
 
   }
-  xyz(id:any){
+  allRow(id:any){
       this.router.navigate(['card/',id])
     }
 
 
     star(data: any) {
       
-      this.service.updateFavorites(data).subscribe((d: any) => {
+      this.service.updateFavorites(data).subscribe()
```

Comment:
reload should come inside subscribe only

Reply:
resolved